src/features/elementfeature.jl

```diff
@@ -65,13 +65,16 @@ function ElementFeatureDescriptor(
     logspaced::Bool = default_log(feature_name, lookup_table),
     categorical::Bool = default_categorical(feature_name, lookup_table),
 )
-    colnames = names(lookup_table)
-    @assert feature_name in colnames && "Symbol" in colnames "Your lookup table must have a column called :Symbol and one with the same name as your feature to be usable!"
-
```

Comment:
Why is this assertion no longer required?

Reply:
Because this (outer) constructor eventually calls the (inner) constructor, which does that check itself!

Reply:
Yes, but before it calls the inner constructor, it calls `get_bins`, which actually requires the `feature_name` to be legit, right?

Reply:
Ahhh fair point. In that case, do you think we should just put the check inside `get_bins`?

Reply:
Yes, I think that would be better, considering one could always call `get_bins` directly